src/search/compound_processing.rs

```diff
@@ -152,6 +152,10 @@ pub fn standardize_mol(romol: &ROMol) -> eyre::Result<ROMol> {
     let te = TautomerEnumerator::new();
     let canon_taut = te.canonicalize(&parent_rwmol.to_ro_mol())?;
     let neutralized_canon = neutralize_atoms(&canon_taut)?;
+
+    // Validate
+    let _ = ROMol::from_smiles(&neutralized_canon.as_smiles())
+        .map_err(|_| eyre::eyre!("Canonicalization failed validation"))?;
```

Comment:
Is this `from_smiles` now in a hot path? How aften do we need to validate neutralized canonical tautomers?

Reply:
We would now be validating them every time. To be safe, I vote we validate everything for now, and them optimize once we have a better sense of all possible failing molecules.

That said, if we become confident that mainly smiles with isotopic hydrogens fail, we could potentially add a `isotopic_hs: bool` argument to `standardize_mol()`. Then we can either perform some special handling for those isotopic hydrogen(s) or at least validate the final molecule in those cases.